core/templates/components/question-directives/question-player/skill-mastery-modal.template.html

```diff
@@ -3,9 +3,9 @@ <h3 ng-if="userIsLoggedIn"> Learning Tips Based on Your Skill Mastery </h3>
 </div>
 
 <div class="modal-body">
-  <skill-mastery-viewer skill-id="skillId"
-                        mastery-change="masteryChange"
-                        ng-if="userIsLoggedIn">
+  <skill-mastery-viewer [skill-id]="skillId"
+                        [mastery-change]="masteryChange"
+                        *ngIf="userIsLoggedIn">
```

Comment:
Let's keep it as `ng-if`, both `*ngIf` and `ng-if` will work here but to keep some sort of consistency with the other syntaxes in the file, I think we can leave this as ng-if. The only change we should make in other files should be just the `[]`. For us, it is clear what this is going on here but there could be many first time angular contributors who could get confused with what changes to make.

Reply:
Done